### DISCUSSION TITLE
Bertweet supported models

### DISCUSSION THREAD

**Discussion by igorkulev** [ThilinaRajapakse/simpletransformers#1263] [Q&A]
Hello,

I am experimenting with NER, more concretely, the Bertweet model (I saw that simpletransformers supports this model), and I noticed that vinai/bertweet-large does not work. Can you please tell me whether this a bug or this model is not supported? I have tested the other betrtweet models: vinai/bertweet-base, vinai/bertweet-covid19-base-cased, and vinai/bertweet-covid19-base-uncased, and they work properly.

Thank you in advance and I would like to use this opportunity to thank you for the great job you are doing: simpletransformers is an awesome library!
Best regards,
Igor

**Comment by ThilinaRajapakse** [comment]
I think the issue probably comes from [here](https://github.com/ThilinaRajapakse/simpletransformers/blob/e9108b274b51fe069046539dec15b31fe5c79fd7/simpletransformers/ner/ner_model.py#L349). 

```python
        if model_name in [
            "vinai/bertweet-base",
            "vinai/bertweet-covid19-base-cased",
            "vinai/bertweet-covid19-base-uncased",
        ]:
            self.tokenizer = tokenizer_class.from_pretrained(
                model_name,
                do_lower_case=self.args.do_lower_case,
                normalization=True,
                **kwargs,
            )
        else:
            self.tokenizer = tokenizer_class.from_pretrained(
                model_name, do_lower_case=self.args.do_lower_case, **kwargs
            )
```

A quick fix would be to just add the name of the model you want to use here. I will change this logic later to check for strings matching any Bertweet model.

**Reply by weihungtseng** [reply]
Hi,

I add the name of the model "vinai/bertweet-large" in 
```
if model_name in [
    "vinai/bertweet-base",
    "vinai/bertweet-covid19-base-cased",
    "vinai/bertweet-covid19-base-uncased",
]:
```

like that:
```
if model_name in [
    "vinai/bertweet-base",
    "vinai/bertweet-covid19-base-cased",
    "vinai/bertweet-covid19-base-uncased",
    "vinai/bertweet-large"
]:
```
, but it still not work, is there have other solution?

Warm regards, 
Mr. Tseng
